docs/guides/mssqlserver/volume-expansion/volume-expansion.md

```diff
@@ -0,0 +1,264 @@
+---
+title: MSSQLServer Volume Expansion
+menu:
+  docs_{{ .version }}:
+    identifier: mssqlserver-volume-expansion-guide
+    name: MSSQLServer Volume Expansion
+    parent: ms-volume-expansion
+    weight: 20
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# MSSQLServer Volume Expansion
+
+This guide will show you how to use `KubeDB` Ops-manager operator to expand the volume of a MSSQLServer.
+
+## Before You Begin
+
+- You need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Now, install KubeDB cli on your workstation and KubeDB operator in your cluster following the steps [here](/docs/setup/README.md). Make sure install with helm command including `--set global.featureGates.MSSQLServer=true` to ensure MSSQLServer CRD installation.
+
+- To configure TLS/SSL in `MSSQLServer`, `KubeDB` uses `cert-manager` to issue certificates. So first you have to make sure that the cluster has `cert-manager` installed. To install `cert-manager` in your cluster following steps [here](https://cert-manager.io/docs/installation/kubernetes/).
+
+- You must have a `StorageClass` that supports volume expansion.
+
+- You should be familiar with the following `KubeDB` concepts:
+  - [MSSQLServer](/docs/guides/mssqlserver/concepts/mssqlserver.md)
+  - [MSSQLServerOpsRequest](/docs/guides/mssqlserver/concepts/opsrequest.md)
```

Comment:
link doesn't work

Reply:
It will work when this pr merged: https://github.com/kubedb/docs/pull/703